packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
+  final Color? backgroundHeaderColor;
```

Comment:
Additionally, many of these new attributes only seem to relate to [StepperType.horizontal] mode. This in itself is fine, but it should be spelled out in each attribute's docstring.